Fix screenshots in features of mpd (#916)
plone/training: docs/mastering-plone/features.md

You can stop the frontend any time using {kbd}`ctrl c`.

> While developing it's not necessary to restart the frontend unless you are adding a new file.

Reviewer: ```suggestion
While developing it's not necessary to restart the frontend unless you're adding a new file.
```

> Login to your new site with "admin" and password "secret".

Reviewer: Inline literals are easier to copy-paste without quotes.

```suggestion
Login to your new site with `admin` and password `secret`.
```

```{figure} _static/frontpage_volto_logged_in.png
```

### Change ports

#### backend

> If you want Plone to listen on port 9080 instead of the default 8080, open the file `backend/instance.yml` in your favorite editor.

Reviewer: MyST syntax for semantic markup.

```suggestion
If you want Plone to listen on port 9080 instead of the default 8080, open the file {file}`backend/instance.yml` in your favorite editor.
```

Therefore this page has the id "news" which we can see as part of its url `http://localhost:3000/news`.

A page is folderish.

> To inspect its contained items, we change to '/contents' by clicking the folder icon.

Reviewer: ```suggestion
To inspect its contained items, we change to `/contents` by clicking the folder icon.
```
